core/controllers/base_test.py

```diff
@@ -279,11 +279,23 @@ def test_alphabetic_i18n_keys(self):
         for filename in filenames:
             with open(os.path.join(os.getcwd(), 'assets', 'i18n', filename),
                       mode='r') as f:
-                next_line = next(f)
-                for line in f:
-                    line = line.replace("{", "")
-                    line = line.replace("}", "")
-                    self.assertGreater(next_line, line)
+                lines = f.readlines()
+                if lines[0] == '{\n':
+                    lines.remove(lines[0])
+                if lines[-1] == '}\n':
+                    lines.pop()
+
+                string_lines = ''.join(lines)
```

Comment:
@seanlip . Thanks a lot for your help. I couldn't think of placing it this way and also sorry for the commented-out code. It was around 2 a.m here and I was just very tired when I was committing the above code.